src/shared/src/flightphase.ts

```diff
@@ -1,10 +1,18 @@
 export enum FmgcFlightPhase {
-    Preflight,
-    Takeoff,
-    Climb,
-    Cruise,
-    Descent,
-    Approach,
-    GoAround,
-    Done,
+    PREFLIGHT,
+    TAKEOFF,
+    CLIMB,
+    CRUISE,
+    DESCENT,
+    APPROACH,
+    GOAROUND,
+    DONE,
```

Comment:
**style:** we generally use UpperCamelCase enum member names in the codebase

Reply:
Hah ok I had only seen https://github.com/flybywiresim/a32nx/blob/master/src/shared/src/autopilot.ts before, will change! and also in the other PR where I touch the file above.